bundles/AdminBundle/Controller/Searchadmin/SearchController.php

```diff
@@ -482,6 +482,14 @@ public function quicksearchAction(Request $request, EventDispatcherInterface $ev
 
                 if ($element instanceof Asset) {
                     $data['iconCls'] .= ' pimcore_icon_' . \Pimcore\File::getFileExtension($element->getFilename());
+                 } elseif ($element instanceof DataObject\Concrete) {
+                    // Show custom on base class (not variants) if it exists
+                    if ($element->getType() === DataObject::OBJECT_TYPE_OBJECT && $element->getClass()->getIcon()) {
```

Comment:
I think we should use the same concept here as we do for the tree, see: https://github.com/pimcore/pimcore/blob/2f18a0f26380c56b609d937b40d2050c8d1a3e1a/bundles/AdminBundle/Controller/Traits/AdminStyleTrait.php#L37